src/app/pages/send/ordinal-inscription/send-inscription-choose-fee.tsx

```diff
@@ -40,28 +40,31 @@ export function SendInscriptionChooseFee() {
   if (isLoadingReview) return <LoadingSpinner />;
 
   return (
-    <BaseDrawer title="Choose fee" isShowing enableGoBack onClose={() => navigate(-1)}>
-      <BitcoinChooseFee
-        amount={createMoney(0, 'BTC')}
-        feesList={
-          <BitcoinFeesList
-            feesList={feesList}
-            onChooseFee={previewTransaction}
-            onValidateBitcoinSpend={onValidateBitcoinFeeSpend}
-            onSetSelectedFeeType={(value: BtcFeeType | null) => setSelectedFeeType(value)}
-            selectedFeeType={selectedFeeType}
-          />
-        }
-        isLoading={isLoading}
-        isSendingMax={false}
-        onChooseFee={previewTransaction}
-        onSetSelectedFeeType={(value: BtcFeeType | null) => setSelectedFeeType(value)}
-        onValidateBitcoinSpend={onValidateBitcoinFeeSpend}
-        recipient={recipient}
-        recommendedFeeRate={recommendedFeeRate}
-        showError={showInsufficientBalanceError}
-        maxRecommendedFeeRate={feesList[0]?.feeRate}
-      />
-    </BaseDrawer>
+    <>
```

Comment:
Reading this code and I wonder if the spinner comes from the check for `isLoadingReview` above that doesn't complete? 


```
async function previewTransaction({ feeRate, feeValue, time, isCustomFee }: OnChooseFeeArgs) {
    try {
      setIsLoadingReview(true);
      await reviewTransaction(feeValue, time, { feeRate, recipient, inscription }, isCustomFee);
    } finally {
      setIsLoadingReview(false);
    }
  }
```

Reply:
It would make sense as then it would just return and show the `LoadingSpinner` and no modal / `Outlet`

Reply:
that was it, thx pete